specification/ai/OpenAI.Assistants/runs/models.tsp

```diff
@@ -202,20 +247,45 @@ model CreateAndRunThreadOptions {
   @doc("The ID of the assistant for which the thread should be created.")
   assistantId: string;
 
-  @doc("The details used to create the new thread.")
+  @doc("The details used to create the new thread. If no thread is provided, an empty one will be created.")
   thread?: AssistantThreadCreationOptions;
 
+  #suppress "@azure-tools/typespec-azure-core/no-nullable" "OpenAI uses explicit nullability, distinct from optionality"
   @doc("The overridden model that the assistant should use to run the thread.")
   @clientName("overrideModelName", "csharp")
-  `model`?: string;
+  `model`?: string | null;
 
+  #suppress "@azure-tools/typespec-azure-core/no-nullable" "OpenAI uses explicit nullability, distinct from optionality"
   @doc("The overridden system instructions the assistant should use to run the thread.")
   @clientName("overrideInstructions", "csharp")
-  instructions?: string;
+  instructions?: string | null;
 
+  #suppress "@azure-tools/typespec-azure-core/no-nullable" "OpenAI uses explicit nullability, distinct from optionality"
   @doc("The overridden list of enabled tools the assistant should use to run the thread.")
   @clientName("overrideTools", "csharp")
-  tools?: ToolDefinition[];
+  tools?: ToolDefinition[] | null;
+
+  /**
+   * If `true`, returns a stream of events that happen during the Run as server-sent events,
+   * terminating when the Run enters a terminal state with a `data: [DONE]` message.
+   */
+  @added(ServiceApiVersions.v2024_02_15_preview)
+  stream?: boolean;
 
   ...OptionalNullableMetadata;
 }
+
+#suppress "@azure-tools/typespec-azure-core/no-nullable" "OpenAI uses explicit nullability, distinct from optionality"
+@doc("Submit the results for the requested tool calls from the service.")
+@added(ServiceApiVersions.v2024_02_15_preview)
+model SubmitToolOutputsOptions {
```

Comment:
Why is this a model? It only seem to be used as a spread source....

Reply:
Agreed--in this case, given it's just two parameters with little likelihood of major expansion in count, I don't think we even need to alias it. I just pushed a commit that merges the contents of the former model directly into the route.

Reply:
> Why is this a model?

The OpenAPI naming to  TSP naming seems to follow the pattern `....Request` -> `...Options`  so in order to have better name parity I thought I'd extract it into a model. I added this from a maintainability perspective, but I agree the value added is tenuous at best.